.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,33 @@
+name: Bug Report
+description: File a bug report
```

Comment:
This will ideally be covered by the org .GitHub repo

Reply:
Gotcha, I'll leave it in here for now to idle until the logistics of that are flushed out (unless they already are 👀 )